docs/multi-manager-setup.md

```diff
@@ -0,0 +1,157 @@
+# High availability in Swarm
+
+In Docker Swarm, the **swarm manager** is responsible for the entire cluster and manages the resources of multiple `docker` hosts at scale. If the swarm manager dies, you must create a new one and deal with an interruption of service.
+
+The *High Availability* feature of Docker Swarm allows a swarm to handle gracefully the failover of a manager instance. Using this feature, you can create a single **primary manager** instance and multiple **replica** instances.
+
+A primary manager is the main point of contact with the Docker Swarm cluster. You can create and talk to replica instances. Requests issued on a replica are automatically proxied to the primary manager. If the primary manager fails, a replica takes away the lead. In this way, you always keep a point of contact with the cluster.
```

Comment:
I don't think "You can create and talk to replica instances." adds very much here, given that the previous paragraph mentions that multiple replicas can be created (but deferring to @moxiegirl in case she had a special reason here)